src/main/java/com/thoughtworks/pacman/ui/screens/IntroScreen.java

```diff
@@ -1,28 +1,130 @@
 package com.thoughtworks.pacman.ui.screens;
 
-import java.awt.Dimension;
+import java.awt.*;
 import java.awt.Graphics2D;
 import java.awt.Image;
-import java.awt.event.KeyEvent;
+import java.awt.event.*;
+import java.awt.Color;
+import java.awt.Rectangle;
+
+import java.awt.font.TextLayout;
+
 
 import com.thoughtworks.pacman.core.Game;
 import com.thoughtworks.pacman.ui.ImageLoader;
 import com.thoughtworks.pacman.ui.Screen;
 
+
 public class IntroScreen implements Screen {
-    static final Image TITLE_SCREEN_IMAGE = ImageLoader.loadImage(Screen.class, "titleScreen.jpg");
+    static final Image TITLE_SCREEN_IMAGE = ImageLoader.loadImage(Screen.class, "deletedText.jpg");
 
     private final Dimension dimension;
     private boolean startGame;
+    private State currentStatePLAY = State.RELEASED__PLAY;
+    private State currentStateSETTINGS = State.RELEASED__SETTINGS;
+    private State currentStateQUIT = State.RELEASED__QUIT;
+    private Color main;
+	private Color hover;
+    
+    private Rectangle playClickBox;
+    private Rectangle settingsClickBox;
+    private Rectangle exitClickBox;
+    
 
     public IntroScreen(Game game) {
         this.dimension = game.getDimension();
+        main = new Color(255, 255, 0);
+        hover = new Color(156, 156, 2);
+        playClickBox = new Rectangle(180, 310, 100, 30);
```

Comment:
Bu sayıların manası nedir? Bunları manalı isimleri olan constantlarda tanımlayıp kullanmak mümkün mü?

Reply:
Bunu dikkate almamışsın. Güncelle bu kod bölgesini

Reply:
@cagdasgerede hocam tam olarak ne yapmami istediginizi anlamadim. Zaten Colorlar'in aciklamasini yorum satirinda yapmistim.